api/src/pcapi/core/users/factories.py

```diff
@@ -29,24 +30,171 @@
 from . import models
 
 
-class BeneficiaryImportStatusFactory(BaseFactory):
+# This file is experiencing a renaissance. It's a good thing.
+# Bellow are the factories that have been deemed correct.
+# You will find the legacy ones at the bottom of the file, waiting to be improved or ruthlessly deleted.
+
+
+class BaseUserFactory(BaseFactory):
+    """
+    Generates a user as if it was created by the account creation form.
```

Comment:
```suggestion
    Generates a user as if it were created by the account creation form.
```
C'est du subjonctif, if I'm not mistaken